docs/source/nlp/distillation.rst

```diff
@@ -0,0 +1,58 @@
+.. _megatron_distillation:
+
+Distillation
+==========================
+
+Knowledge Distillation (KD)
+--------------------------------
+
+KD involves using information from an existing trained model to train a second (usually smaller, faster) model, thereby "distilling" knowledge from one to the other.
+
+Distillation has two primary benefits: potentially faster convergence and/or higher end accuracy than traditional training.
```

Comment:
I think we can just say "faster convergence and higher end accuracy than traditional training"